### PR TITLE
PODS-10211:Migrated Charge A folder to twirl

### PR DESCRIPTION
**Pull request by dynamicwave4**

### CODE REVIEW

**Review by rambabu-posa — APPROVED** [review]

File: app/controllers/chargeA/ChargeDetailsController.scala [hunk 1]
```diff
@@ -53,21 +51,14 @@ class ChargeDetailsController @Inject()(override val messagesApi: MessagesApi,
                                         val controllerComponents: MessagesControllerComponents,
                                         deleteChargeHelper: DeleteChargeHelper,
                                         config: FrontendAppConfig,
-                                        renderer: Renderer)(implicit ec: ExecutionContext)
+                                        view : ChargeDetailsView)(implicit ec: ExecutionContext)
     extends FrontendBaseController
     with I18nSupport
     with NunjucksSupport {
```
Comment (rambabu-posa) [inline app/controllers/chargeA/ChargeDetailsController.scala:57]: sorry forgot to comment on this one
Please remove this `with NunjucksSupport` and import

File: app/controllers/YearsController.scala [hunk 2]
```diff
@@ -41,7 +43,8 @@ class YearsController @Inject()(
                                 val controllerComponents: MessagesControllerComponents,
                                 renderer: Renderer,
                                 config: FrontendAppConfig,
-                                schemeService: SchemeService
+                                schemeService: SchemeService,
+                                yearsView: YearsView
                             )(implicit ec: ExecutionContext)
                                 extends FrontendBaseController
                                 with I18nSupport
```
Comment (rambabu-posa) [inline app/controllers/YearsController.scala:50]: sorry forgot to comment on this one
Please remove this with NunjucksSupport and import

Comment (dynamicwave4) [reply app/controllers/YearsController.scala]: done

File: app/controllers/chargeA/ChargeDetailsController.scala (hunk 1 above)
Comment (dynamicwave4) [reply app/controllers/chargeA/ChargeDetailsController.scala]: done

**Pull request merged**